packer/openhpc_extravars.yml

```diff
@@ -1 +1,2 @@
 workaround_ansible_issue_61497: yes # extravars files can't be empty
+appliances_mode: "build"
```

Comment:
Can we move this to environments/common/inventory/group_vars/builder/defaults.yml please? Just so there is only one place to look for things. And also remove the `"` just for consistency, barewords like this are fine.